modules/distribution/src/main/scripts/fluo

```diff
@@ -67,6 +70,7 @@ function validate_app {
   if [[ $APP = *"-h"* ]]; then
     print_usage
   fi
+  java -cp "$FLUO_LIB_DIR/*:$FLUO_LIB_DIR/logback/*" io.fluo.cluster.util.ValidateAppName $APP
```

Comment:
It takes about .34 sec.  I think this is OK but I want to refactor a little to make sure it's only run by 'new' & 'init' commands.  All other command are later on in lifecycle and don't need to validate.